File: sub-model/src/main/java/org/github/base/service/impl/SysMenuServiceImpl.java
package org.github.base.service.impl;

import org.github.base.entity.SysMenuEntity;
import org.github.base.mapper.ISysMenuMapper;
import org.github.base.service.ISysMenuService;
import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 * sys_menu_table 服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2019-03-24
 */
@Service
public class SysMenuServiceImpl extends ServiceImpl<ISysMenuMapper, SysMenuEntity> implements ISysMenuService {}

File: sub-mysql-sccore-service/src/main/java/org/github/mysql/sccore/base/service/impl/SysMenuServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.sccore.base.service.impl;

import org.github.mysql.sccore.base.entity.SysMenuEntity;
import org.github.mysql.sccore.base.mapper.ISysMenuMapper;
import org.github.mysql.sccore.base.service.ISysMenuService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * 菜单管理 服务实现类
 *
 * @author JYD_XL
 */
@Service
open class SysMenuServiceImpl : ServiceImpl<ISysMenuMapper, SysMenuEntity>(), ISysMenuService
